c7n/ufuncs/s3crypt.py

```diff
@@ -45,7 +45,7 @@ def process_key_event(event, context):
     processor = EncryptExtantKeys(config)
     for record in event.get('Records', []):
         bucket = record['s3']['bucket']['name']
-        key = {'Key': record['s3']['object']['key'],
+        key = {'Key': urllib.unquote_plus(record['s3']['object']['key'].encode('utf8')),
```

Comment:
also rather unconditional unquote'ing, the simple check if the call is nesc is `if '%' in key`

Reply:
And also `+` - This is what `urllib.unquote_plus` is doing internally; Are you worried about performance?

Reply:
nah fine as is. was just recently in that part of the stdlib and saw thats what it was triggering on.